test/integration/prometheus_endpoint_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule MongoosePush.API.PrometheusEndpointTest do
+  use ExUnit.Case, async: false
+  use AssertEventually, timeout: 5000, interval: 10
+
+  alias MongoosePush.Support.API, as: Tools
+
+  @url "/v1/notification/f534534543"
+
+  setup do
+    Tools.reset(:fcm)
+    :ok
+  end
+
+  @tag :skip
+  test "Metrics can be correctly received from Prometheus" do
```

Comment:
@rslota @NelsonVides, can this test case be enabled now when Phoenix is finally used instead of Maru?

Reply:
@michalwski I've already made that change here: https://github.com/esl/MongoosePush/pull/159/files#diff-74898c36c774be8df73c05fbcd46d647L14